domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/displaycontextparameter/DisplayContextParameterType.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.ccd.definition.store.domain.displaycontextparameter;
+
+import com.google.common.base.Strings;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public enum DisplayContextParameterType {
+    DATETIMEENTRY,
+    DATETIMEDISPLAY,
```

Comment:
```suggestion
    DATETIME_DISPLAY,
```

Reply:
The value entered on import is DATETIMEENTRY/DATETIMEDISPLAY so this keeps it consistent 👌